postgres/pgxv4/postgres.go

```diff
@@ -54,6 +58,17 @@ type pgDriver struct {
 //
 // "host=my-project:us-central1:my-db-instance user=myuser password=mypass"
 func (p *pgDriver) Open(name string) (driver.Conn, error) {
+	var dbURI string
+	var ok bool
+
+	p.mu.RLock()
+	dbURI, ok = p.dbURIs[name]
```

Comment:
I worry a bit about using `name` as the key, simply because another caller could very well re-order the parameters and produce a different key. Is there a better key that we could use here?

Reply:
I'm pretty sure that connection string stays more or less immutable in most of the cases, and it's more or less driver specific ... I would suggest not change this at least in this MR.